src/context/NoteContext.js

```diff
@@ -0,0 +1,40 @@
+import { createContext, useContext, useEffect, useState } from "react";
+
+import axios from "axios";
+
+const NoteContext = createContext({});
+
+const NoteProvider = ({ children }) => {
+  const [notes, setNotes] = useState([]);
+
+  useEffect(() => {
+    (async () => {
```

Comment:
You can separate this logic to authContext and store it in the state variable

Reply:
i didn't got you why would we need to shift it to auth context